site/gatsby-site/cypress/e2e/apps/submitted.cy.js

```diff
@@ -419,4 +419,44 @@ describe('Submitted reports', () => {
 
     cy.get('@modal').should('not.exist');
   });
+
+  maybeIt(
+    'Does not allow promossion of submission if developers, deployers or harmed parties is missing.',
```

Comment:
"Submit" becomes "submi**ss**ion," but alas: "promote" becomes "promo**t**ion." This is one of the [ghoti](https://en.wikipedia.org/wiki/Ghoti) sounds in our magnificent orthographic system.

Reply:
@lmcnulty ouf! My bad, I actually know this! Fixed